.github/workflows/playwright.yml

```diff
@@ -0,0 +1,49 @@
+name: Playwright tests on pr
+on:
+  pull_request:
+    branches: [main]
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'packages/**'
+      - './**'
+      - '.github/workflows/playwright.yml'
+      - 'package.json'
+      - 'pnpm-lock.yaml'
+      - 'pnpm-workspace.yaml'
+      - 'turbo.json'
+      - 'biome.json'
+
+  workflow_dispatch:
+
+jobs:
+  playwright-tests:
+    name: 'Playwright Tests'
+    env:
+      PLAYWRIGHT_TEST_BASE_URL: 'http://localhost:5173' # vite default port
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    steps:
+      - name: 'Checking Out Code'
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 'Installing Dependencies'
+        uses: ./.github/actions/install
+
+      - name: Install Playwright Browsers
+        run: pnpm --filter frontend-design-poc install:browsers
+
+      - name: Run Playwright tests
+        run: pnpm --filter frontend-design-poc test:playwright
+
+      - uses: actions/upload-artifact@v3
```

Comment:
![Screenshot 2024-02-23 at 13 56 44](https://github.com/digdir/dialogporten-frontend/assets/213777/010fc9ca-469d-43a9-a6d3-ae52523cd138)

This stage does not seem to work?
Maybe the `path` is wrong? Probably needs to be `packages/frontend-design-poc/playwright-report/` or something.